CMakeLists.txt

```diff
@@ -192,33 +186,34 @@ if (USE_LIBCUDF)
     #       instead of hardcoding paths to includes and to libraries, like below
     target_include_directories(meta-cudf-parser-1 PUBLIC
                                /usr/local/include
-                               /usr/local/include/libcudf/libcudacxx
+                               $ENV{CONDA_PREFIX}/include
     )
     target_include_directories(meta-cudf-parser-1 AFTER PUBLIC
-                               /opt/conda/envs/rapids/include
+        $ENV{CONDA_PREFIX}/include
```

Comment:
universal instead of hardfixed

Reply:
also /opt/conda/envs/rapids doesn't exist in a default conda installation of rapids